### PR TITLE
Clarify release instructions

### PR DESCRIPTION
**Pull request by sashaCher**
### Desired Outcome

Some release instructions during last release procedure were not clear for me.
This PR clarifies some of them.

### Implemented Changes

Some release instructions were changed.

### Connected Issue/Story

Part of 12.4 release effort [ONYX-11563](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-11563)

### Definition of Done

- [X] Release process is more clear

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [X] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [X] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [X] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [X] There are no security aspects to these changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,7 +108,8 @@ to publish new versions of the image to public registries:
    - Commit these changes- `Bump version to x.y.z` is an acceptable commit
      message - and open a PR for review.
 1. Merge the version bump PR, and tag the repo with the new version.
-   - Ensure you are working from a current clone of the main branch
+   - Merge PR into main branch
+   - Ensure you are working from the up to date main branch
```
Comment (tzheleznyak) [inline CONTRIBUTING.md:112]: There is no enforcement of rebase in this repository ?

Comment (sashaCher) [reply CONTRIBUTING.md:112]: What does it mean?
Once you're merging PR you need to switch back to main locally and to pull it.

Comment (tzheleznyak) [reply CONTRIBUTING.md:112]: I meant that github won't let merge if branch is not rebased. Like it in some project for example conjur oss. Seems but its not related so ok this instruction can stay.

**Review by tzheleznyak — APPROVED** [review]
LGTM

**Comment by foxefj-cyber** [comment]
@sashaCher - does this need to be part of the 12.5 release?

**Comment by sashaCher** [comment]
@foxefj-cyber Feels me irrelevant after automatic release process implementation, closing...

**Pull request closed without merging**